API.md

```diff
@@ -307,10 +307,15 @@ OpenSCD core sets the following CSS variables on the plugin:
 
 ```css
 * {
-  --oscd-primary: var(--oscd-theme-primary, #2aa198);
-  --oscd-secondary: var(--oscd-theme-secondary, #6c71c4);
-  --oscd-error: var(--oscd-theme-error, #dc322f);
-
+  --oscd-accent-yellow: var(--oscd-theme-accent-yellow, #b58900);
+  --oscd-accent-orange: var(--oscd-theme-accent-orange, #cb4b16);
+  --oscd-accent-red: var(--oscd-theme-accent-red, #dc322f);
+  --oscd-accent-magenta: var(--oscd-theme-accent-magenta, #d33682);
+  --oscd-accent-violet: var(--oscd-theme-accent-violet, #6c71c4);
+  --oscd-accent-blue: var(--oscd-theme-accent-blue, #268bd2);
+  --oscd-accent-cyan: var(--oscd-theme-accent-cyan, #2aa198);
+  --oscd-accent-green: var(--oscd-theme-accent-green, #859900);
```

Comment:
```suggestion
  --oscd-yellow: var(--oscd-theme-yellow, #b58900);
  --oscd-orange: var(--oscd-theme-orange, #cb4b16);
  --oscd-red: var(--oscd-theme-red, #dc322f);
  --oscd-magenta: var(--oscd-theme-magenta, #d33682);
  --oscd-violet: var(--oscd-theme-violet, #6c71c4);
  --oscd-blue: var(--oscd-theme-blue, #268bd2);
  --oscd-cyan: var(--oscd-theme-cyan, #2aa198);
  --oscd-green: var(--oscd-theme-green, #859900);
```

I'd personally prefer these shorter variable names. What do you think about them?

Explanation:

With the hitherto provided variable names (primary, secondary, error, base2, text-font, etc.) my goal was to make them as short and snappy as possible for the benefit of plugin authors.

Had we created a full CSS design system instead of just theming tokens for a single app, we should have come up with some modular naming scheme like `oscd-color-primary-100` `oscd-color-base2-100` `oscd-font-text-body-regular` or something like that. I thought that level of verbosity wasn't needed here.